engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubConnection.java

```diff
@@ -136,4 +143,11 @@ private URIBuilder appendToPath(final URIBuilder uriBuilder, final String pathSe
 
         return uriBuilder.setPath(pathSegment);
     }
+
+    private class DataHubRequestRetryHandler extends DefaultHttpRequestRetryHandler {
+        DataHubRequestRetryHandler() {
+            super(3, false,
+                    Arrays.asList(InterruptedIOException.class, UnknownHostException.class, SSLException.class));
```

Comment:
Any advantage to creating a subclass that just calls the constructor of the super class?

I see a bit in hiding the details, but since it is only used privately, the benefits to that are pretty limited.

OTOH, it's not like it's hard to understand, so nothing lost from this... Mostly just made me curious :)